pkg/limayaml/defaults.go

```diff
@@ -535,6 +535,16 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 
 	caCerts := unique(append(append(d.CACertificates.Certs, y.CACertificates.Certs...), o.CACertificates.Certs...))
 	y.CACertificates.Certs = caCerts
+
+	if y.UseDefaultDependencyResolution == nil {
+		y.UseDefaultDependencyResolution = d.UseDefaultDependencyResolution
+	}
+	if o.UseDefaultDependencyResolution != nil {
+		y.UseDefaultDependencyResolution = o.UseDefaultDependencyResolution
+	}
+	if y.UseDefaultDependencyResolution == nil {
+		y.UseDefaultDependencyResolution = pointer.Bool(true)
```

Comment:
Wondering this should be rather inverted to `SkipDefaultDependency` with the default value `false`